cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2383,7 +2398,7 @@ function start-kube-controller-manager {
     params+=" --pv-recycler-pod-template-filepath-hostpath=$PV_RECYCLER_OVERRIDE_TEMPLATE"
   fi
   if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
-    RUN_CONTROLLERS="nodelifecycle"
+    RUN_CONTROLLERS="serviceaccount,serviceaccount-token,nodelifecycle"
```

Comment:
why do we need this now and didn't need it for insecure?

Reply:
not need functionality wise as we tested it already in POC. but better to keep them running for better apple to apple comparison for perf runs.

Reply:
the api server rbac routines will be triggered in secure mode. token and account service controller is required there. no need for insecure mode.